keyboards/checkerboards/quark/rules.mk

```diff
@@ -17,8 +17,8 @@ BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
 RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
 AUDIO_ENABLE = no           # Audio output
 UNICODE_ENABLE = yes        # Unicode
+ENCODER_ENABLE = yes	    # Enable Rotary Encoders
 
-LAYOUTS = ortho_4x12 planck_mit
```

Comment:
is there a reason for removing this?

Reply:
I haven't seen it on any other boards so I didn't think it was needed. What's its purpose?

Reply:
It adds community support.  

Eg, if you try `qmk compile -kb checkerboards/quark -km drashna`, it would use the files in `/layouts/community/ortho_4x12/drashna`. 

eg, it allows users to maintain support for a large number of keyboards without having to have dozens of keymaps, basically.

Reply:
TIL - thank you